packages/base-styles/README.md

```diff
@@ -0,0 +1,37 @@
+# Base styles
+
+Base scss utilities and variables for WordPress.
+
+## Installation
+
+Install the module
+
+```bash
+npm install @wordpress/base-styles --save-dev
+```
+
+## Use
+
+### Scss utilities and variables
+
+In your application's scss file, include styles like so:
+
+```scss
+@import "~@wordpress/base-styles/colors";
+@import "~@wordpress/base-styles/variables";
+@import "~@wordpress/base-styles/mixins";
+@import "~@wordpress/base-styles/breakpoints";
+@import "~@wordpress/base-styles/animations";
+@import "~@wordpress/base-styles/z-index";
+```
+
+### Post-CSS themes
```

Comment:
```suggestion
### PostCSS themes
```

Reply:
Might need to be "PostCSS color schemes"tho? See https://github.com/WordPress/gutenberg/pull/17883#discussion_r335365862